### PR TITLE
CHANGE(grafana): Provision 2 new diagnostic dashboards

### PR DESCRIPTION
**Pull request by fatpat**
##### SUMMARY
- diag_host_details: to view system details about one single host
- diag_hosts_comparison: to compare metrics between hosts

 ##### ISSUE TYPE
- Feature Pull Request

### CODE REVIEW

**Review by vdombrovski — CHANGES_REQUESTED** [review]
Please import dashboards using provided tools/retriever.py script

**Comment by fatpat** [comment]
> Please import dashboards using provided tools/retriever.py script

done

File: files/diag_host_details.json [hunk 1]
```diff
@@ -0,0 +1,2178 @@
...
+            "y": 13,
+            "w": 12,
+            "h": 8,
+            "x": 0
+          },
+          "links": [],
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.oioswift\"}) by (dimension)",
```
Comment (vdombrovski) [inline files/diag_host_details.json:1710]: Hardcoded NS

File: files/diag_host_details.json [hunk 2]
```diff
@@ -0,0 +1,2178 @@
...
+            "y": 13,
+            "w": 12,
+            "h": 8,
+            "x": 12
+          },
+          "links": [],
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.meta1\"}) by (dimension)",
```
Comment (vdombrovski) [inline files/diag_host_details.json:1798]: Hardcoded NS

File: files/diag_host_details.json [hunk 3]
```diff
@@ -0,0 +1,2178 @@
...
+            "y": 21,
+            "w": 12,
+            "h": 8,
+            "x": 0
+          },
+          "links": [],
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.rawx\"}) by (dimension)",
```
Comment (vdombrovski) [inline files/diag_host_details.json:1887]: Hardcoded NS

File: files/diag_host_details.json [hunk 4]
```diff
@@ -0,0 +1,2178 @@
...
+            "y": 21,
+            "w": 12,
+            "h": 8,
+            "x": 12
+          },
+          "links": [],
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(aggr:netdata_web_log_http_request_methods:sum{instance=~\"$host\",chart2=\"RAKUTEN.meta2\"}) by (dimension)",
```
Comment (vdombrovski) [inline files/diag_host_details.json:1976]: Hardcoded NS

File: files/diag_host_details.json [hunk 5]
```diff
@@ -0,0 +1,2178 @@
...
+        "y": 4,
+        "w": 12,
+        "h": 8,
+        "x": 0
+      },
+      "links": [],
+      "timeFrom": null,
+      "targets": [
+        {
+          "expr": "sum(netdata_apps_cpu_system_cpu_time___average{chart=\"apps.cpu_system\",family=\"cpu\",instance=~\"$host$\"}) by (dimension)",
```
Comment (vdombrovski) [inline files/diag_host_details.json:1263]: Wrong percentage display (should be percent 0.0-1.0)

File: files/diag_hosts_comparison.json [hunk 6]
```diff
@@ -0,0 +1,2190 @@
...
+            "y": 4,
+            "w": 12,
+            "h": 8,
+            "x": 0
+          },
+          "links": [],
+          "timeFrom": null,
+          "targets": [
+            {
+              "expr": "sum(netdata_apps_cpu_system_cpu_time___average{chart=\"apps.cpu_system\",family=\"cpu\",instance=~\"$host$\"}) by (dimension)",
```
Comment (vdombrovski) [inline files/diag_hosts_comparison.json:1270]: Wrong percentage display

**Review by vdombrovski — CHANGES_REQUESTED** [review]

**Comment by vdombrovski** [comment]
Besides required fixes to merge, this PR only adds to dashboards to maintain without giving much more new information (all info can be found in other dashboards).

I am against integrating it into release until we don't have proper QA auto-validating dashboard queries.

**Comment by cdelgehier** [comment]
ping @fatpat

**Pull request closed without merging**